third_party/zlib/google/zip.h

```diff
@@ -158,6 +158,14 @@ bool UnzipWithFilterCallback(const base::FilePath& zip_file,
                              const FilterCallback& filter_cb,
                              bool log_skipped_files);
 
+#if defined(IN_MEMORY_UPDATES)
+// An overload for a zip represented as a string.
+bool UnzipWithFilterCallback(const std::string& zip_str,
```

Comment:
I'd rather omit it because I think it's redundant since IN_MEMORY_UPDATES implies that it's a Cobalt customization. And I think we'll likely update the IN_MEMORY_UPDATES references to STARBOARD (or, better yet, USE_COBALT_CUSTOMIZATIONS) anyway once the feature's complete (see comment in starboard/evergreen/shared/platform_configuration/BUILD.gn). Let me know if you feel strongly though and I can add the STARBOARD guards.